server/db/interface.go

```diff
@@ -0,0 +1,186 @@
+// This code is available on the terms of the project LICENSE.md file,
+// also available online at https://blueoakcouncil.org/license/1.0.0.
+
+package db
+
+import (
+	"github.com/decred/dcrdex/server/account"
+	"github.com/decred/dcrdex/server/market/types"
+	"github.com/decred/dcrdex/server/order"
+)
+
+type DEXArchivist interface {
+	OrderArchiver
+}
```

Comment:
A complete DEXArchivist will be composed of several different interfaces.  Starting with `OrderArchiver`

Reply:
Might as well just say this in the doc string.